Makefile

```diff
@@ -1,41 +1,19 @@
 all: ci rhel
 .PHONY: all
 
+OFFICIAL = keys/verifier-public-key-redhat-release keys/verifier-public-key-redhat-beta-2 stores/store-openshift-official-release stores/store-openshift-official-release-mirror
+
 # The CI system uses the OpenShift CI public key and verifies it against a bucket on GCS.
-ci:
-	echo "# Release verification against OpenShift CI keys" > \
-		manifests/0000_90_cluster-update-keys_configmap.yaml
-	oc create configmap release-verification \
-			--from-file=keys/verifier-public-key-openshift-ci \
-			--from-file=stores/store-openshift-ci-release \
-			--dry-run -o yaml | \
-		oc annotate -f - release.openshift.io/verification-config-map= \
-			-n openshift-config-managed --local --dry-run -o yaml	>> \
-		manifests/0000_90_cluster-update-keys_configmap.yaml; \
-	echo "  namespace: openshift-config-managed" >> \
-		manifests/0000_90_cluster-update-keys_configmap.yaml
+ci: keys/verifier-public-key-openshift-ci stores/store-openshift-ci-release $(OFFICIAL)
```

Comment:
Are either of these steps called by a presubmit?

Reply:
If not, I'd like to see that presubmit added.

Reply:
Filed openshift/release#10064, although I'm not clear on the best way to run in a `cli` image.  Rehearsals there should let me know if my first guess doesn't work, anyway :p.